dynawo/sources/API/PAR/test/TestParametersSet.cpp

```diff
@@ -215,8 +215,7 @@ TEST(APIPARTest, ParametersSetGetReferences) {
     ++nbReferences;
   ASSERT_EQ(nbReferences, 3);
 
-  ParametersSet::reference_const_iterator itRef = parametersSet->cbeginReference();
-  ASSERT_EQ(itRef->get()->getName(), "ref3");
+  ASSERT_EQ(ref3->getName(), "ref3");
```

Comment:
Please do not modify the unit tests. This one was made to test cbeginReference

Reply:
There are a few things I don't understand :
cbeginReference() is called elsewhere in the unit test so it is actually tested even if I delete this line
I don't understand what "begin" means here. An unordered_map has actually no beginning. I had to change this unit test because the std version worked locally but had a different result on the CI.

Reply:
parametersSet is not a unordered_map